lib/Path/Tiny.pm

```diff
@@ -1534,19 +1534,33 @@ sub mkpath {
 
     path("foo.txt")->move("bar.txt");
 
-Move the current path to the given destination path using Perl's
-built-in L<rename|perlfunc/rename> function. Returns the result
-of the C<rename> function (except it throws an exception if it fails).
+Moves the current path to the given destination using L<File::Copy>'s
+C<move> function. Upon success, returns the C<Path::Tiny> object for the
+newly moved file.
 
-Current API available since 0.001.
+Current API available since 0.124. Prior versions used Perl's
```

Comment:
Just to follow convention, this should be the last paragraph in the pod for the method.

Reply:
indeed, fixed